### PR TITLE
[DO NOT MERGE] Added support for mina-verify-packaged-fork-config

### PR DESCRIPTION
**Pull request by dkijania**
Publish mina-verify-packaged-fork-config in CI

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/deb-builder-helpers.sh [hunk 1]
```diff
@@ -368,6 +368,22 @@ build_archive_migration_deb () {
...
 
+##################################### VERIFY PACKAGED FORK CONFIG PACKAGE ##########################################
+build_verify_packaged_fork_config_deb () {
+
+  LEDGER_VALIDATION_DEB=mina-verify-packaged-fork-config${DEB_SUFFIX}
+
+  echo "------------------------------------------------------------"
+  echo "--- Building verify packaged fork config deb"
+
+  create_control_file "$LEDGER_VALIDATION_DEB" "${DEAMON_DEPS} ,${ARCHIVE_MIGRATION_DEPS} ,${SUGGESTED_DEPS}" 'Berkeley Fork Config Verification Tool' "mina-logproc, ${MINA_DEB_NAME}"
```
Comment (emberian) [inline scripts/deb-builder-helpers.sh:379]:
```suggestion
  create_control_file "$LEDGER_VALIDATION_DEB" "${DAEMON_DEPS} ,${ARCHIVE_MIGRATION_DEPS} ,${SUGGESTED_DEPS}" 'Berkeley Fork Config Verification Tool' "mina-logproc, ${MINA_DEB_NAME}"
```

**Review by emberian — CHANGES_REQUESTED** [review]

**Comment by dkijania** [comment]
This is ok for docker and I will update it. However, it is not feasible for debian package. Which should include only script.

**Review by emberian — APPROVED** [review]

**Comment by dkijania** [comment]
After internal discussion this is not needed as app is already in daemon debian

**Pull request closed without merging**